src/CurrentRouteInterface.php

```diff
@@ -28,4 +28,21 @@ public function getRoute(): ?RouteParametersInterface;
      * @return UriInterface|null The current URI.
      */
     public function getUri(): ?UriInterface;
+
+    /**
+     * Returns the current route parameters.
+     *
+     * @return array
+     */
+    public function getParameters(): array;
```

Comment:
If we have a route with path `/{foo}/{bar}` and it matched with `/hello/world` request URI then:
 - parematers are `['foo', 'bar']`
 - arguments are `['foo' => 'hello', 'bar' => 'world']`

I hope the code is consistent with the terminology

Reply:
PSR-7 uses "params" so, I think, we should be consistent with it.

Reply:
It is not the same case. User has already configured the parameters in the route like custom function.
PSR-7 has nothing to do with it at all.